CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 ## [0.1] - 2024-06-05
 ### Added
-- MeltingTemperature enum
+- `MeltingTemperature` enum
```

Comment:
Could you please let me know when we add `` and when we don't?

Reply:
Usually, we use backticks for specific objects, like variables, files, and so on.
For example:

1. `params.py` 🆗 
2. `MeltingTemperature` 🆗
3. `complement` method 🆗 
4. `Unit` test 👎